catalog/to-consul/syncer.go

```diff
@@ -475,40 +474,3 @@ func (s *ConsulSyncer) init() {
 		s.initialSync = make(chan bool)
 	}
 }
-
-func (s *ConsulSyncer) checkAndCreateNamespace(ns string) error {
-	// Check if the Consul namespace exists
-	namespaceInfo, _, err := s.Client.Namespaces().Read(ns, nil)
-	if err != nil {
-		return err
-	}
-
-	// If not, create it
-	if namespaceInfo == nil {
-		var aclConfig api.NamespaceACLConfig
-		if s.CrossNamespaceACLPolicy != "" {
-			// Create the ACLs config for the cross-Consul-namespace
-			// default policy that needs to be attached
-			aclConfig = api.NamespaceACLConfig{
-				PolicyDefaults: []api.ACLLink{
-					{Name: s.CrossNamespaceACLPolicy},
-				},
-			}
-		}
-
-		consulNamespace := api.Namespace{
-			Name:        ns,
-			Description: "Auto-generated by a Catalog Sync Process",
```

Comment:
These descriptions used to be different for each component that created the namespace but now they're all going to be "auto-generated by consul-k8s". I think that's a fine tradeoff for deleting all the duplicated code.